src/messaging/ExchangeContext.cpp

```diff
@@ -254,7 +254,9 @@ ExchangeContext::ExchangeContext(ExchangeManager * em, uint16_t ExchangeId, Sess
     SetDropAckDebug(false);
     SetAckPending(false);
     SetMsgRcvdFromPeer(false);
-    SetAutoRequestAck(true);
+
+    // Do not request Ack for multicast
+    SetAutoRequestAck(!session.GetGroupId().HasValue());
```

Comment:
Would prefer a `IsGroupSession` accessor on `session`.  As things stand, it's not clear to me whether `GetGroupId()` would be true or not for MCSP....  But the spec concept of "group session" is pretty pinned-down.

Reply:
Added